wordcloud/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.conf.urls import patterns, url
+
+from .views import wordcloud
+
+urlpatterns = patterns(
+    '',
+    url(r'^wordcloud/$', wordcloud, name='wordcloud'),
```

Comment:
The renaming to wordcloud makes sense, but I'd keep `tagcloud` as an alias in `urls.py` just for the transition - we can remove it later.